test/fonts/test_font_remap.py

```diff
@@ -1,36 +1,38 @@
 from pathlib import Path
 
 from fpdf import FPDF
-from fpdf.fonts import SubsetMap
+
+# from fpdf.fonts import SubsetMap
 from test.conftest import assert_pdf_equal
 
 HERE = Path(__file__).resolve().parent
 
-
-def test_subset_map():
-    subset_map = SubsetMap(range(0, 1024, 2))
-    assert len(subset_map.dict()) == 512
-
-    for i in range(0, 1024, 2):
-        assert i % 2 == 0
-        assert i == subset_map.pick(i)
-
-    for i in range(1023, 512, -2):
-        assert subset_map.pick(i) % 2 == 1
-    assert len(subset_map.dict()) == 512 + 256
-
-    for i in range(1, 1000, 2):
-        assert subset_map.pick(i) % 2 == 1
-
-    assert len(subset_map.dict()) == 1024
-
-    subset_dict = subset_map.dict()
-    for i in subset_dict:
-        for j in subset_dict:
-            if i != j:
-                assert subset_dict[i] != subset_dict[j]
-            else:
-                assert subset_dict[i] == subset_dict[i]
+# SubsetMap was rewritten in a class with many different functions
+# TO DO : Rewrite this test
```

Comment:
Should this be done before merging this PR?

Reply:
I'll remove this test.
We have plenty of other tests with TTF fonts - specially the first 999 characters ones that can only pass if the subset is identical to the reference files.

Reply:
OK